src/main/java/frc/robot/RobotContainer.java

```diff
@@ -153,6 +153,8 @@ public class RobotContainer {
   PathConstraints hpPathConstraints = new PathConstraints(4.0, 3.5, 8.42, 12.8876585);
 
   PathConstraints scorePathConstraints = new PathConstraints(3.0, 2.5, 8.0, 10.0);
+  PathConstraints pluckPathConstraints = new PathConstraints(2.0, 2.0, 8.0, 10.0);
```

Comment:
I was suggesting constraining the path in pathplanner, it's not bad to have this as well though.